Formula/n3dr.rb

```diff
@@ -0,0 +1,20 @@
+class N3dr < Formula
+  desc "Nexus3 Backup and Recovery tool"
+  homepage "https://n3dr.releasesoftwaremoreoften.com/"
+  url "https://github.com/030/n3dr/archive/refs/tags/6.0.5.tar.gz"
+  sha256 "0b50539ea4d2858ec8a4f6b8f6776a45a634db26b89dce35d5e527305ed8738a"
+  license "MIT"
+  depends_on "go" => :build
+
+  def install
+    mkdir "cmd/n3dr" do
+      system "go", "build", "-ldflags", "-X main.Version=refs/tags/#{version}", "-o", "n3dr"
+    end
+    bin.install "cmd/n3dr/n3dr" => "n3dr"
+  end
+
+  test do
+    assert_match "n3dr version refs/tags/#{version}",
+      shell_output("n3dr --version")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula